src/components/ProductDetailsMain/ProductDetailsMain.component.tsx

```diff
@@ -0,0 +1,113 @@
+import { useContext, useEffect, useState } from 'react';
+import { StatesContext } from '../../store/GlobalStateProvider';
+// eslint-disable-next-line max-len
+import { Line } from '../base/Line/Line.component';
+import cn from 'classnames';
+import { Price } from '../base/Price/Price.component';
+import { getProductsSummary } from '../../api/products';
+import { ProductSummary } from '../../types/ProductSummary';
+import { CardButtons } from '../base/CardButtons/CardButtons.component';
+import { SpecsMini } from '../base/SpecsMini/SpecsMini.component';
+
+type Colors = {
+  [key: string]: string;
+};
+
+const colors: Colors = {
+  black: '#4C4C4C',
+  spaceblack: '#565C6A',
+  green: '#5b7E65',
+  yellow: '#EED96C',
+  white: '#F0F0F0',
+  purple: '#CACDF5',
+  red: '#B1081B',
+  gold: '#F4E4D1',
+  sierrablue: '#AAC6E3',
+  graphite: '#96959D',
+  silver: '#AAAAAF',
+  spacegray: '#B6ADA7',
+  'space gray': '#B6ADA7',
+  blue: '#6B89B3',
+  'sky blue': '#7CACD0',
+  'rose gold': '#FDD8D3',
+  rosegold: '#FDD8D3',
+  starlight: '#E8E0D5',
+  pink: '#FD889A',
+  midnightgreen: '#626B64',
+  midnight: '#31373F',
+  coral: '#F19584',
+};
```

Comment:
also I think there are some variables/colors/infos like these that could be comprised in a folder like public/variables or something